codex/codex.nim

```diff
@@ -55,12 +56,16 @@ type
   EthWallet = ethers.Wallet
 
 proc bootstrapInteractions(
-    config: CodexConf,
-    repo: RepoStore
-): Future[Contracts] {.async.} =
+    s: CodexServer
+): Future[void] {.async.} =
   ## bootstrap interactions and return contracts
   ## using clients, hosts, validators pairings
   ##
+  let
+    config = s.config
+    repo = s.repoStore
+
+  s.codexNode.clock = SystemClock()
```

Comment:
Can we set this later as the `else` branch of the `if config.validator or config.persistence:`?

Then the assignments to node.clock are in one place, and a bit easier to reason about.